### ISSUE TITLE
strong param arrays

### ISSUE THREAD

**Issue by cngonzalez** [learn-co-curriculum/forms-and-basic-associations-rails#11]
Hi there,

Sorry for the bother. This lesson provides an example for making an array (post_ids) a strong parameter via the category_params method. This example may be incorrect. According to the rails strong parameter documentation on github (https://github.com/rails/strong_parameters), arrays must be explicitly called out in a permit method.

In other words, students may run into problems trying to emulate 

```
def category_params
    params.require(:category).permit(:name, :post_ids)
  end
end
```

when the following is closer to what the documentation says:

```
 def category_params
    params.require(:category).permit(:name, {:post_ids=> []})
  end
end
```

(I mention it because I had problems with this in the next lab!)

Thanks for all your hard work,
C

**Comment by pletcher** [comment]
Thanks for being a part of the Learn community!

Keep in mind that this space is for specific issues related to the content of this lesson.

If you're reporting on the content of this lesson, it'd be super helpful if you could include a stack trace of any errors that you're seeing.

If you'd like to report a bug about the Learn IDE, the Ask a Question form, opening a file, or the lights not working on Learn, please report a bug directly on https://learn.co. (You can find the bug report form by clicking on your avatar in the upper right, and then selecting "Report A Bug".)

We want to respond as quickly as possible and be as helpful as possible, so please be sure to include steps to reproduce the issue (as best you can remember) and any pertinent details about your environment. (Are you using the Learn IDE? Coding from the ISS?) We'll figure out what's up and get back to you ASAP.

**Comment by pletcher** [comment]
Hi! You're right -- we'll get this fixed up ASAP. Thanks for reporting it!

@AnnJohn, not to add more steps than necessary, but who's the best person to assign this to?

**Comment by gj** [comment]
Fixed in #13. Thanks @cngonzalez !
